src/frontend/src/common/icons/IconFilters.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
should we take out these attrs and add them to the style.ts instead? (that's how i've been styling the other svgs)

Reply:
My instinct says to leave them in the svg. What is the benefit of pulling them out into a style sheet?